src/cloud-code/Parse.Cloud.js

```diff
@@ -8,40 +8,205 @@ function getClassName(parseClass) {
   return parseClass;
 }
 
+/** @namespace
+ * @name Parse
+ * @description The Parse SDK.
+ *  see [api docs](https://docs.parseplatform.org/js/api) and [guide](https://docs.parseplatform.org/js/guide)
+ */
+
+/** @namespace
+ * @name Parse.Cloud
+ * @memberof Parse
+ * @description The Parse Cloud Code SDK.
+ */
+
 var ParseCloud = {};
+/**
+ * Defines a Cloud Function.
+ *
+ * **Available in Cloud Code only.**
+ * **Starting parse-server 3.0.0, all cloud hooks and functions are asynchronous**
+ * **All response objects are removed and replaced by promises / async/await based resolution**
+ *
+ * @static
+ * @memberof Parse.Cloud
+ * @param {String} name The name of the Cloud Function
+ * @param {Function} data The Cloud Function to register. This function can be an async function and should take one parameter a {@link Parse.Cloud.FunctionRequest}.
+ */
 ParseCloud.define = function(functionName, handler, validationHandler) {
   triggers.addFunction(functionName, handler, validationHandler, Parse.applicationId);
 };
 
+/**
+ * Defines a Background Job.
+ *
+ * **Available in Cloud Code only.**
+ *
+ * @method job
+ * @name Parse.Cloud.job
+ * @param {String} name The name of the Background Job
+ * @param {Function} func The Background Job to register. This function can be async should take a single parameters a {@link Parse.Cloud.JobRequest}
+ *
+ */
 ParseCloud.job = function(functionName, handler) {
   triggers.addJob(functionName, handler, Parse.applicationId);
 };
 
+/**
+ *
+ * Registers an before save function.
```

Comment:
remove the leading line.  `an before` should be `a before`

Reply:
All of that were already in the JS SDK btw :)

Reply:
no doubt.